node/src/command.rs

```diff
@@ -252,9 +249,7 @@ pub fn run() -> Result<()> {
 		Some(Subcommand::ExportGenesisHead(cmd)) => {
 			let runner = cli.create_runner(cmd)?;
 			runner.sync_run(|config| {
-				construct_benchmark_partials!(config, |partials| {
-					cmd.run(partials.client)
-				})
+				construct_benchmark_partials!(config, |partials| cmd.run(partials.client))
```

Comment:
This code block is different from the one in `base-parachain` because we need it for dynamically support the two runtimes `devnet` and `testnet`
```rs
macro_rules! construct_benchmark_partials {
	($config:expr, |$partials:ident| $code:expr) => {
		match $config.chain_spec.runtime() {
			Runtime::Devnet => {
				let $partials = new_partial::<pop_runtime_devnet::RuntimeApi>(&$config)?;
				$code
			},
			Runtime::Testnet => {
				let $partials = new_partial::<pop_runtime_testnet::RuntimeApi>(&$config)?;
				$code
			},
		}
	};
}
```